client/lib/notification-settings-store/toggle-state.js

```diff
@@ -1,44 +1,68 @@
 /** @format */
+/**
+ * Internal dependencies
+ */
+import { find, get, without } from 'lodash';
+
+const toggleInStream = ( streamName, stream, setting ) => ( {
+	[ streamName ]: {
+		...stream,
+		[ setting ]: ! get( stream, setting ),
+	},
+} );
+
+const toggleInDevice = ( devices, deviceId, setting ) => {
+	const device = find( devices, ( { device_id } ) => device_id === parseInt( deviceId, 10 ) );
```

Comment:
Nit:
```js
const deviceIdNum = parseInt( deviceId, 10 );
const device = find( devices, { device_id: deviceInNum } );
```
parses the string into a number only once and uses `lodash.find` shorthand for matching a property value.

Reply:
Good spot :)